sites/public/src/ListingView.tsx

```diff
@@ -201,6 +205,13 @@ export const ListingView = (props: ListingProps) => {
       </div>
       <div className="w-full md:w-2/3 md:mt-3 md:hidden md:mx-3 border-gray-400 border-b">
         <ListingUpdated listing={listing} />
+        <div className="mx-4">
+          <ApplicationSection
+            listing={listing}
+            preview={props.preview}
+            internalFormRoute="/applications/start/choose-language"
```

Comment:
Is this a required prop? Wouldn't it drop the user into a 404 page?

Reply:
Sorry this was me playing with something in the mobile view.  Removed.

Reply:
That's a good point though, does this show up in the mobile view? Maybe we could add another hide prop to ApplicationSection, or maybe Exygy is willing to consider this logic inside this component.